examples/basic-typescript/package.json

```diff
@@ -8,18 +8,18 @@
     "icons": "npx @svgr/cli --typescript --replace-attr-values \"#000=currentColor\" --out-dir generated/icons -- public/icons"
   },
   "dependencies": {
-    "@makeswift/runtime": "0.14.0",
+    "@makeswift/runtime": "0.0.0-app-router-support-20240326193332",
     "@radix-ui/react-accordion": "^1.1.2",
     "@radix-ui/react-tabs": "^1.0.4",
     "clsx": "^2.0.0",
-    "next": "13.2.1",
+    "next": "14.1.4",
     "react": "18.2.0",
     "react-dom": "18.2.0"
   },
   "devDependencies": {
     "@trivago/prettier-plugin-sort-imports": "^4.2.0",
     "@types/node": "17.0.37",
-    "@types/react": "^18.2.0",
+    "@types/react": "18.2.60",
```

Comment:
https://linear.app/makeswift/issue/ENG-4902/figure-out-why-latest-version-of-typesreact-breaks-lazy-component

I will fill this in at some point.

The Bigcommerce catalyst example isn't seeing this problem because it has a lock file.